ui/app/src/state/epics/audiencesPanel.ts

```diff
@@ -25,7 +25,8 @@ import {
   SET_ACTIVE_TARGETING_MODEL,
   SET_ACTIVE_TARGETING_MODEL_COMPLETE,
   setActiveTargetingModelComplete as setActiveTargetingModelCompleteAction,
-  setActiveTargetingModelFailed
+  setActiveTargetingModelFailed,
+  setActiveTargetingModelFailed as setActiveTargetingModelFailedAction
```

Comment:
Why import twice with different names?

Reply:
This was a mistake, removed the duplicate